canonical-web-and-design/ubuntu.com
# Changes for Discourse v4

## Done

- Changes for Discourse v4

## QA

- Check /tutorials works as expected
- Check documentation pages are working as well as expected
